src/EnergyPlus/ReportSizingManager.cc

```diff
@@ -3425,6 +3455,10 @@ namespace ReportSizingManager {
 
         } else if (SizingType == CoolingCapacitySizing) {
             if (coilSelectionReportObj->isCompTypeCoil(CompType)) {
```

Comment:
Coil inlet temp is now reported to Coil Sizing Report while capacity is being calculated IF RequestSizing actually sets this data.